packages/commerce-sdk-react/src/test-utils.tsx

```diff
@@ -23,6 +23,7 @@ export const DEFAULT_TEST_HOST = 'http://localhost:8888'
 
 export const DEFAULT_TEST_CONFIG = {
     proxy: `${DEFAULT_TEST_HOST}/mobify/proxy/api`,
+    slasProxy: `${DEFAULT_TEST_HOST}`,
```

Comment:
This is how I am consuming the new proxy set in ssr.js. As mentioned in other comments, it is currently separate from `/mobify/proxy/api`